int32 CremalheiraHorizontal
int32 CremalheiraVertical